src/components/sbb-calendar/sbb-calendar.tsx

```diff
@@ -57,6 +56,26 @@ interface CalendarKeyboardNavigationParameters {
   verticalOffset: number;
 }
 
+export interface Day {
+  value: string;
+  dayValue: string;
+  monthValue: string;
+  yearValue: string;
+}
+
+export interface Month {
+  value: string;
+  longValue: string;
+  monthValue: number;
+}
+
+export interface Weekday {
+  long: string;
+  narrow: string;
+}
+
+export type CalendarView = 'day' | 'month' | 'year';
+
 export const events = {
   dateSelected: 'date-selected',
 };
```

Comment:
We have several occurrences of "Date | string | number". would it mabye be worth to create a named type for that?

Reply:
Done, but not sure about naming;
'DateLike' - 'AcceptedDateTypes'  'AcceptedDateLike' ?

Reply:
good question, I think I could live with all of them ;). @kyubisation what do you think?

Reply:
`SbbDateLike` (the winner?) sounds fine to me 👍